lib/src/signed_video_h26x_common.c

```diff
@@ -583,10 +583,10 @@ remove_epb_from_sei_payload(h26x_nalu_t *nalu)
   nalu->tlv_start_in_nalu_data = nalu->payload + UUID_LEN;
   nalu->tlv_size = nalu->payload_size - UUID_LEN;
   nalu->reserved_byte = *nalu->tlv_start_in_nalu_data;
-  nalu->start_of_sei = ((nalu->reserved_byte & (1 << 6)) != 0);
   nalu->tlv_start_in_nalu_data++;  // Move past the |reserved_byte|.
   nalu->tlv_size -= 1;  // Exclude the |reserved_byte| from TLV size.
   nalu->with_epb = (nalu->reserved_byte & 0x80);  // Hash with emulation prevention bytes
+  nalu->is_start_of_stream_sei = (nalu->reserved_byte & 0x40);  // Parse flags from reserved byte
```

Comment:
The comment came out wrong. This comment should be placed before all parsed flags. There will be more flags coming up in the future.